scripts/common/addon.sh

```diff
@@ -144,9 +144,14 @@ function addon_outro() {
 
             printf "\n\t${GREEN}${prefix}upgrade.sh | sudo bash -s${common_flags}${NC}\n\n"
         fi
-        printf "Press enter to proceed\n"
-        prompt
 
+        if prompts_can_prompt ; then
+            echo "Press enter to proceed"
+            prompt
+        else
+            echo "TODO: what to do here"
+            # logWarn "Upgrade script must be run on all remote nodes"
```

Comment:
I'm not sure what to do in this case. There is no way to automate this currently unless it continues. Should there be a warning? Or should we add an additional flag?

Reply:
We want to be able to support full automation if desirable, so preferably a flag.

Reply:
Added properties Kurl.IgnoreRemoteLoadImagesPrompt and Kurl.IgnoreRemoteUpgradePrompt to bypass prompts for automation.

Reply:
Cool, thanks. Let's make sure we doc those

Reply:
https://github.com/replicatedhq/kurl.sh/pull/413